.fixtures.yml

```diff
@@ -2,18 +2,13 @@ fixtures:
   repositories:
     stdlib:
       repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
-      ref: '4.6.0'
+      ref: '5.2.0'
     concat:
       repo: 'https://github.com/puppetlabs/puppetlabs-concat.git'
-      ref: '2.2.1'
-    common:
-      repo: 'https://github.com/ghoneycutt/puppet-module-common.git'
-      ref: 'v1.4.1'
-    firewall:
-      repo: 'https://github.com/puppetlabs/puppetlabs-firewall.git'
-      ref: '1.9.0'
+      ref: '5.2.0'
     sshkeys:
-      repo: 'git://github.com/puppetlabs/puppetlabs-sshkeys_core.git'
-      ref: '1.0.1'
+      repo: 'https://github.com/puppetlabs/puppetlabs-sshkeys_core.git'
+      ref: '1.0.2'
+      puppet_version: '>= 6.0.0'
   symlinks:
```

Comment:
with recent [puppetlabs_spec_helper](https://github.com/puppetlabs/puppetlabs_spec_helper/commit/b62449bc388cfc62d06630eac9f70b1b7492999e#diff-1237b5f9023710b4e56fc7899c71f98dR35) symlinks should not be necessary.